blog-posts/thoughs-on-collaborative-work.md

```diff
@@ -0,0 +1,81 @@
+leadboxText: Are you excited about getting the most out your team work? Would you like to work with great professionals? Then join us!
+
+
+---
+
+Most of us developers, despite being more or less aware of the benefits of doing more work together, work on our own most of the time. Why? What can we do about it? Let's dive into it!
+
+## Is it just about programming?
+
+Short answer, no, it’s not. As software developers we don’t just write code, we also play a crucial role in actively shaping, challenging and clarifying requirements. And this happens in many different flavours: emails, issue tracking systems, chat tools, team meetings, inter-team meetings... Every area of our work can benefit if we approach it in a more collaborative way.
+
+## The good, the bad and the ugly. Or is it just good?
+
+What is the greatest hesitance when thinking about working more with other colleagues? Well, most of us think it is generally a waste of time, that it is an inefficient way to work, that there is no time for that. But if we dig deeper, we will find two strong causes. First, what I call "the developer pride". We think we are great if we solve problems on our own, if we do it without help, but if we ask for help it means we are not good enough. Second, shame. Often times we feel like what we do is not good enough to be shared. The idea of someone watching you work, the typos you do, the uncertainty with which you approach problems, etc. is frightening for most of us. Literally “stage fright”.
+
+These fears are common both in junior and in senior developers, but the truth is that we are all learning all the time, and the more we interact with others, the more we learn. I am not an exception, but I decided to open myself more to collaborative work for three main reasons:
+
+- I was not happy with the amount and quality of collaborative work in my teams,
+- I consider these beliefs mostly as prejudices, and
+- Every time I was approaching work in a more collaborative way, even though feeling unnatural, it showed to be quite powerful.
+
+Don't get me wrong, I am not talking about overdoing it. It is important to keep the balance, working many hours per day in groups is much more tiring, challenging and also more inefficient than just working on your own. So what we really want is to find the range where collaborative work produces a positive impact in our working life. In order for that to happen, it is crucial to accumulate experiences and get a feeling of which collaboration paradigm is better to use for each use case. Daily meeting? Knowledge sharing session? Second opinion? Think of a toolbox for team collaboration, this is exactly what is offered to you. Every person, every situation, every topic is different, that's what makes this journey exciting and why you want to have a variety of options and adapt each time.
```

Comment:
I like the toolbox thought here and the way you sell the idea to pursue collaboration work.